tests/data/dummy_skill/dummy_subpackage/foo.py

```diff
@@ -18,8 +18,32 @@
 # ------------------------------------------------------------------------------
 
 """This module is in a skill sub-package (for testing purposes)."""
+from aea.skills.base import Behaviour
 
 
 def bar():
     """A bar function."""
     return 42
+
+
+class DummyBehaviour(Behaviour):
```

Comment:
WE should also add a handler class here and not give it a file path, just to have a richer test case.